models/spleen_deepedit_annotation/docs/README.md

```diff
@@ -21,6 +21,13 @@ Input: 3 channels CT image - one channel representing clicks for each segment (i
 
 Output: 2 channels: Label 1: spleen; Label 0: everything else - This depends on the dictionary "label_names" defined in train.json and inference.json ("label_names": {"spleen": 1, "background": 0},)
 
+## Model Performance
+
+The achieved Dice score on the validation set is:
+
+Spleen: 0.959
+
+
 ## commands example
 Execute training:
 
```

Comment:
here needs to provide a command to do the validation

Please also update the metadata.json for `changelog` and `version`, thanks!